tests/application/app/DummyManagerReturnsImpl.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App;
+
+use Illuminate\Support\Manager;
+
+/**
+ * @mixin DummyDriverContract
+ */
```

Comment:
You shouldn't need this mixin, this is the whole reason why we have the extension

Reply:
Ah, yes, I must have added it by automatism =_=